packages/blade/src/components/Input/TextInput/TextInput.tsx

```diff
@@ -241,16 +241,19 @@ const _TextInput: React.ForwardRefRenderFunction<BladeElementRef, TextInputProps
 
   const trailingDropdown =
     trailing && getComponentId(trailing as React.ReactElement) === 'Dropdown' ? trailing : null;
-
+  // we need to look into name of component and check if it 's and icon or a dropdown
   const _leadingIcon: IconComponent | undefined =
```

Comment:
odd, how was it working before? I don't think we have componentId set in icons as "Icon"